templates/3_Pixie_Cluster_Cells.ipynb

```diff
@@ -297,7 +297,9 @@
    "cell_type": "markdown",
```

Comment:
<p>I think it would be better to move this to right after the remapping (so the results are saved even if someone doesn't run this visualization block).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/999/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>